EventListener/ContactClientSubscriber.php

```diff
@@ -204,7 +204,7 @@ public function onTimelineGenerate(ContactClientTimelineEvent $event)
                         ],
                         'eventType'       => $eventTypeName,
                         'extra'           => [
-                            'logs'                => $log,
+                            'logs'                => strip_tags($log),
```

Comment:
Isn't $log in this context always a json encoded output?

I thought $row['message'] is what needed strip_tags.